designs/dragonphy_top/constraints/configure.yml

```diff
@@ -18,7 +18,7 @@ parameters:
 
   # Period of the main clock in nanoseconds
   # (will be scaled by constr_time_scale)
-  constr_main_per: 0.9
+  constr_main_per: 0.7
```

Comment:
Just wanted to check this is intended, since we had previously discussed increasing 0.7 to 0.9